Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.mm

```diff
@@ -204,8 +204,8 @@ void displayLinkFired(PlatformDisplayID displayID, DisplayUpdate, bool wantsFull
 
     willHandleWheelEvent(wheelEvent);
 
-    ScrollingThread::dispatch([dispatcher = Ref { *this }, wheelEvent, rubberBandableEdges] {
-        dispatcher->scrollingThreadHandleWheelEvent(wheelEvent, rubberBandableEdges);
+    ScrollingThread::dispatch([dispatcher = Ref { *this }, wheelEvent = WebWheelEvent { wheelEvent }, rubberBandableEdges] {
```

Comment:
Do we really need the ` = WebWheelEvent { wheelEvent }` here? It should be identical to the previous code.